validate.js

```diff
@@ -0,0 +1,57 @@
+const bb = require('ssb-bendy-butt')
+const bfe = require('ssb-bfe')
+
+exports.validateSingle = function (contentSection) {
+  if (!Array.isArray(contentSection) || contentSection.length !== 2)
```

Comment:
Another way to write this is: `!(Array.isArray(contentSection) && contentSection.length === 2)`.